### PR TITLE
New package: megacmd-3.2.0

### PR DESCRIPTION
**Pull request by jbgg**

### CODE REVIEW

**Comment by lemmi** [comment]
Context: #7725

**Comment by jbgg** [comment]
On musl system it depens on https://github.com/voidlinux/void-packages/issues/7773

File: srcpkgs/megacmd/template [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+hostmakedepends="base-devel autogen doxygen"
+makedepends="libcurl-devel c-ares-devel libressl-devel crypto++-devel zlib-devel sqlite-devel freeimage-devel readline-devel libsodium-devel pcre-devel pcre2-devel libuv-devel fuse-devel"
+depends="pcre libpcrecpp c-ares libcurl libsodium sqlite freeimage libressl crypto++"
+short_desc="The official client for cloud service mega.nz"
+maintainer="jbgg <jbosco.gg@gmail.com>"
+license="2-clause-BSD"
+homepage="https://mega.nz"
+distfiles="https://github.com/meganz/sdk/archive/v${version}.tar.gz"
+checksum=4f4b607caa80fc7a91ba1cbb882b1c9b903d9b67b59db47fb7ec821e33de29f3
+only_for_archs="x86_64 i686 x86_64-musl i686-musl"
```
Comment (Vaelatern) [inline srcpkgs/megacmd/template]: You close and open and close and open your PRs all over, yet you don't think to remove this line?

If you did discuss this previously, too bad. I'm not digging through the history.

File: srcpkgs/megacmd/template [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Template file for 'megacmd'
+pkgname=megacmd
+version=3.2.0
+revision=1
+wrksrc=sdk-${version}
+build_style=gnu-configure
+make_build_args='LIBTOOLFLAGS="--tag=CXX"'
+hostmakedepends="base-devel autogen doxygen"
+makedepends="libcurl-devel c-ares-devel libressl-devel crypto++-devel zlib-devel sqlite-devel freeimage-devel readline-devel libsodium-devel pcre-devel pcre2-devel libuv-devel fuse-devel"
```
Comment (Vaelatern) [inline srcpkgs/megacmd/template]: Please cap line lengths at 80 chars. All lines that are continued to the next line follow this convention:
no `\` at end of lines
Continued lines begin with a space (not a tab. a space).

File: srcpkgs/megacmd/template (hunk 1 above)
Comment (Vaelatern) [reply srcpkgs/megacmd/template]: more likely you need `nocross` than this. `nocross` should be set to the URL of an example failing build.

**Comment by jbgg** [comment]
It doesn't compile on aarch64 because of libfreeimage (see: https://travis-ci.org/voidlinux/void-packages/jobs/281616641).

**Pull request merged**